README.md

```diff
@@ -1,2 +1,16 @@
-# passport
-The backend of AI4HF Product Passport
+# Passport 
```

Comment:
Refactor the README file and get rid of the outdated information. Further, do not include any details regarding to the frontend, just provide a simple guide to start the application with correct settings. And use code blocks whenever possible.